src/routes/PublicLayoutRouter/PublicLayoutRouter.scss

```diff
@@ -67,14 +67,20 @@
 }
 
 .center-column {
-  padding-bottom: 100px;
   display: flex;
   align-items: center;
   justify-content: center;
 }
 
-.center-column > div {
+.non-map-view {
+  padding-bottom: 100px;
+}
+
+.non-map-view > {
```

Comment:
do we really need both a .map-view and a .non-map-view class? seems like you could just use one or the other and have a default that then gets overridden by the one class being set?

Reply:
I tried doing that; its effectively the center-column styling that is the default. However, there are some other stylings that depend on the center-column class being present, so I couldn't just replace it without replicating all of those as well. This ended up being the least amount of code change